models/Translation/Dao.php

```diff
@@ -115,6 +115,23 @@ public function getAvailableLanguages()
         return $languages;
     }
 
+    /**
+     * Returns a array containing all available domains
+     *
+     * @return array
+     */
+    public function getAvailableDomains()
+    {
+        $domainTables = $this->db->fetchAll("SHOW TABLES LIKE 'translations_%'");
+        $domains = [];
+
+        foreach ($domainTables as $domainTable) {
+            $domains[] = str_replace('translations_', '', current($domainTable));
```

Comment:
```suggestion
            $domains[] = str_replace('translations_', '', $domainTable);
```

Reply:
we just need value here:
![image](https://user-images.githubusercontent.com/5137917/145444866-18cd9447-a089-476f-95e8-4653ae7c95a8.png)

Reply:
Oh, wow! Didn't expect this!

Maybe writing it as follows would make it more clear:

```suggestion
            $domains[] = str_replace('translations_', '', $domainTable[array_key_first($domainTable)]);
```